addon/factory-guy.js

```diff
@@ -379,7 +379,7 @@ class FactoryGuy {
         fixture     = this.buildRaw(assign(args, {buildType: 'make'}));
 
     if (this.isModelAFragment(modelName)) {
-      return fixture;
+      return this.store.createFragment(modelName, fixture);
```

Comment:
I'm assuming that I don't need to `this.store.normalize` the fixture as this has already been done inside `buildRaw`?